docs/UsersGuide/source/BuildingRunningTesting/WE2Etests.rst

```diff
@@ -38,15 +41,30 @@ For convenience, the WE2E tests are currently grouped into the following categor
 * ``grids_extrn_mdls_suites_nco``
    This category of tests ensures that the workflow running in **NCO mode** (i.e., with ``RUN_ENVIR`` set to ``"nco"``) completes successfully for various combinations of predefined grids, physics suites, and input data from different external models. Note that in NCO mode, an operational run environment is used. This involves a specific directory structure and variable names (see :numref:`Section %s <NCOModeParms>`).
 
+* ``ufs_case_studies``
+   This category tests that the workflow running in community mode completes successfully when running cases derived from the `ufs-case-studies repository <https://github.com/dtcenter/ufs-case-studies>`__. 
+
 * ``verification``
    This category specifically tests the various combinations of verification capabilities using METPlus. 
 
-* ``release_SRW_v1``
-   This category was reserved for the official "Graduate Student Test" case for the Version 1 SRW code release.
-
 * ``wflow_features``
    This category of tests ensures that the workflow completes successfully with particular features/capabilities activated.
 
+.. note::
+
+   Users should be aware that some tests assume :term:`HPSS` access. 
+   
+      * ``custom_ESGgrid_Great_Lakes_snow_8km`` and ``MET_verification_only_vx_time_lag`` require HPSS access. 
```

Comment:
I added that `rstprod` access is required. Not sure we need the details on how to request, since it's relevant to a small subset of users and will look like gobbledegook to ppl outside of NOAA/NCAR type orgs. People in these orgs can usually find out from their teams how to request access to whatever they need. I can add the full note if you think it's important tho.

Reply:
The modifications that you have made are perfect.  We don't want to bog down users who might not even have access to WCOSS2, Hera, or Jet.  Resolving the conversation now.